packages/nodes-base/nodes/Google/Sheet/v2/actions/sheet/append.operation.ts

```diff
@@ -12,12 +17,12 @@ export const description: SheetProperties = [
 		options: [
 			{
 				name: 'Auto-Map Input Data to Columns',
-				value: 'autoMapInputData',
+				value: RESOURCE_MAPPING_MODES.AUTO,
 				description: 'Use when node input properties match destination column names',
 			},
 			{
 				name: 'Map Each Column Below',
-				value: 'defineBelow',
+				value: RESOURCE_MAPPING_MODES.MANUAL,//'defineBelow',
 				description: 'Set the value for each destination column',
 			},
 			{
```

Comment:
![image](https://user-images.githubusercontent.com/88898367/227179965-719f9263-f3d8-42fc-9c59-3f40ffa4bab9.png)
changing values in dataMode would break v3